Makefile

```diff
@@ -51,6 +52,7 @@ assets-watch: ## Watch frontend assets (during development)
 .PHONY: clear-cache
 clear-cache: ## Clear caches for specified environment (default: SYMFONY_ENV=dev)
 	$(PHP_RUN) bin/console cache:clear --env=$(SYMFONY_ENV)
+	$(PHP_RUN) bin/console cache:pool:clear $(CACHE_POOL) --env=$(SYMFONY_ENV)
```

Comment:
I would not want to have this running with `clear-cache` command by default. Local installations often do not use redis cache for persistence (when using a local DB), so this will fail in those cases.

We should introduce a second command that includes both cache:clear and cache:pool:clear, e.g. `clear-all-cache` or something similar.

Reply:
@emodric Fixed in https://github.com/netgen/media-site/pull/93/commits/d4bf9dfb9accd4f86753fa4c9c8c6204668fd44b.